assets/js/theme/global/maintenanceMode.js

```diff
@@ -7,17 +7,43 @@ import $ from 'jquery';
 export default function (maintenanceMode = {}) {
     const header = maintenanceMode.header;
     const notice = maintenanceMode.notice;
+    const password = maintenanceMode.password;
```

Comment:
i'm confused - what's to stop a user from putting a breakpoint to see the preview code?